src/components/Dropdown/Dropdown.module.scss

```
@@ -0,0 +1,47 @@
+@import "@config/variables.scss";
+
+.dropdown{
+    cursor: pointer;
+
+    list-style: none;
+
+    color: $grey-color;
+
+    font-family: 'Circular Std';
```

на всякий случай лучше fallback-шрифт указать, типа:
```suggestion
    font-family: 'Circular Std', sans-serif;
```